webservices/resources/dates.py

```diff
@@ -60,24 +58,25 @@ def get(self, **kwargs):
 @doc(description='FEC election dates since 1995.')
 class ElectionDatesView(DatesResource):
 
-    model = models.ElectionDates
+    model = models.ElectionDate
+
     @property
     def date_column(self):
         return self.model.election_date
 
-    filter_fields = {
-        'election_state',
-        'election_district',
-        'election_party',
-        'office_sought',
-        'election_date',
-        'trc_election_type_id',
-        'trc_election_status_id',
-        'update_date',
-        'create_date',
-        'election_year',
-        'pg_date',
-    }
+    filter_multi_fields = [
```

Comment:
Sounds reasonable, although now I'm realizing that I don't know what all the dates and other columns actually refer to here--for example, what's `pg_date`, and why would a user want to filter on it? Do we want to include `trc_election_type_id` and `trc_election_status_id`? At minimum, it seems like we'll need `election_date` and `due_date` above to have range filters, since I would guess we'd need those for user-facing views. Seems reasonable to make all date fields behave the same way.

Anyway, the goal of this patch is to only make sure you're not blocked on this work by n+1 queries, so feel free to add or modify filters as you see fit :smiley: